.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -4,6 +4,13 @@ about: Create a report to help us improve Electron
 
 ---
 
+<!-- To help make it easier for us to investigate your issue, please follow the Contributing Guidelines: https://github.com/electron/electron/blob/master/CONTRIBUTING.md.
+
+This project abides by the following Code of Conduct: https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md
```

Comment:
I kind of like how forge works this into their template:

https://github.com/electron-userland/electron-forge/blame/master/.github/issue_template.md#L15

Do you have any opinion on using that approach?

Reply:
ooh yes!

Reply:
will update imminently 🚀